MANIFEST.in

```diff
@@ -1,9 +1,8 @@
 include wandb/py.typed
 include package_readme.md
 include requirements.txt
-include requirements.sweeps.txt
+include requirements_sweeps.txt
 include wandb/bin/apple_gpu_stats
-include wandb/sweeps/config/schema.json
```

Comment:
@dannygoldstein why was it needed here?

Reply:
because this jsonschema is used to infer sweep parameter types from passed config files in the local controller. it's part of the sweeps package. we won't need to include it in the manifest if we make a requirement instead of vendoring.